ratpack-cassandra-migrate/src/main/java/smartthings/ratpack/cassandra/CassandraMigrationService.java

```diff
@@ -10,7 +10,7 @@
 import smartthings.migration.MigrationParameters;
 import smartthings.migration.MigrationRunner;
 
-@DependsOn(CassandraService.class)
+@DependsOn(Session.class)
```

Comment:
I believe this works as long as the provided implementation extends `@Service`.   It would be good to test that part more though.